pkg/yttlibraryext/all.go

```diff
@@ -10,5 +10,6 @@
 package yttlibraryext
 
 import (
+	_ "github.com/vmware-tanzu/carvel-ytt/pkg/yttlibraryext/math" // include math
```

Comment:
Agreed. Not well-documented: that "extensions" are _only_ intended for modules that may be opt'ed out of because they bring a potentially unwanted dependency for those using `ytt` as a Go module.

In addition to the point of "registration" @cppforlife noted, above, an example of another "standard library" module that uses only Golang dependencies: https://github.com/vmware-tanzu/carvel-ytt/blob/develop/pkg/yttlibrary/ip.go

Reply:
> Not well-documented

It's written just above, I missed it :|

Reply:
> It's written just above, I missed it :|

Given the level of detail that you are addressing, I'd say that signal is not strong enough.

Re-reading that paragraph, it looks like a wall-of-text to me. Even just a little formatting and tightening some word-choice might make a difference.

I'll attempt this in a separate PR.